cmd/support/generate/main.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
I put this in a `cmd` support. This follows the pattern established by the community